examples/Tools/FluentUI.Demo.SampleData/People.cs

```diff
@@ -30,4 +49,104 @@ public static IEnumerable<string> FacesForImg
             yield return IMG_PREFIX_JPG + ResourceReader.EmbeddedPicture("Face8.jpg").ToBase64();
         }
     }
+
+    /// <summary>
+    /// Gets a list of 50 persons with random data.
+    /// </summary>
+    public static IEnumerable<Person> Persons => GeneratePersons(50);
+
+    /// <summary>
+    /// Gets a list of 50 persons with random data and a very long name to validate components.
+    /// </summary>
+    public static IEnumerable<Person> PersonsWithVeryLongName => GeneratePersons(50).Union([VeryLongNamePerson]);
+
+    /// <summary>
+    /// Get a person with a very long name to validate components.
+    /// </summary>
+    public static readonly Person VeryLongNamePerson = new Person(
+        Id: "91",
+        FirstName: "Jean",
+        LastName: "With a very long name to validate components",
+        Male: true,
+        Initials: "JW",
+        BirthDay: new DateTime(1984, 4, 27),
+        ImageUrl: "https://thispersondoesnotexist.com/?id=91",
```

Comment:
Should this not point to one of the embedded images (same for other references further down)?

Reply:
The thispersondoesnotexist.com website contains no arguments for specifying the image to be displayed. Each call generates a new image. This `id` is a “false” argument to inform the browser that the file must be downloaded and not come from the cache.